package.json

```diff
@@ -42,53 +42,108 @@
+                "cph.language.c.Args": {
                     "title": "Compilation flags for .c files",
                     "type": "string",
                     "default": "",
-                    "description": "Space seperated additional flags passed to gcc ( for C) while compiling your file. Example '-O2 -Wall'"
+                    "description": "Space seperated additional flags passed to gcc ( for C ) while compiling your file. Example '-O2 -Wall'"
                 },
-                "cph.argsCpp": {
+                "cph.language.c.SubmissionCompiler": {
+                    "type": "string",
+                    "default": "GNU GCC C11 5.1.0",
+                    "enum": [
+                        "GNU GCC C11 5.1.0"
+                    ],
+                    "description": "The compiler chosen in the drop down during Codeforces submission for c"
+                },
+                "cph.language.cpp.Args": {
                     "title": "Compilation flags for .cpp files",
                     "type": "string",
                     "default": "",
                     "description": "Space seperated additional flags passed to g++ ( for C++ ) while compiling your file. Example '-Wmaybe-uninitialized -std=c++14'"
                 },
-                "cph.argsPython": {
+                "cph.language.cpp.SubmissionCompiler": {
+                    "type": "string",
+                    "title": "check12344566",
+                    "default": "GNU G++17 7.3.0",
+                    "enum": [
+                        "GNU G++17 7.3.0",
+                        "GNU G++14 6.4.0",
+                        "GNU G++11 5.1.0",
+                        "Microsoft Visual C++ 2017",
+                        "Microsoft Visual C++ 2010",
+                        "Clang++17 Diagnostics"
+                    ],
+                    "description": "The compiler chosen in the drop down during Codeforces submission for cpp"
+                },
+                "cph.language.python.Args": {
                     "title": "Compilation flags for Python",
                     "type": "string",
                     "default": "",
                     "description": "Space seperated additional flags passed to python while compiling your file. Example '-Wignore -u'"
                 },
-                "cph.argsRust": {
+                "cph.language.python.SubmissionCompiler": {
+                    "type": "string",
+                    "default": "PyPy 3.6 (7.2.0)",
+                    "enum": [
+                        "PyPy 3.6 (7.2.0)",
+                        "Python 3.7.2",
+                        "PyPy 2.7 (7.2.0)",
+                        "Python 2.7.15"
+                    ],
+                    "description": "The compiler chosen in the drop down during Codeforces submission for python"
+                },
+                "cph.language.python.Command": {
+                    "type": "string",
+                    "default": "python3",
```

Comment:
I have kept the default command for python as `python3` instead of `python` since most users would expect it to use the latest version by default.

Reply:
Cool. The issue is, in Windows, `python` points to 3.x version. In Linux, `python` points to 2.X and `python3` points to 3.x. That's the reason so many people were facing issues. But thats no longer an issue now, as it can be changed here, if needed.

Reply:
Oh I see. Another thing that popped up in my head is that people might not know if such a setting exists right. So, when a python command fails, we can also show a message saying you might want to check out/edit this setting.

Reply:
Yeah, that's a good idea. If you want to, you can make a PR :)